modules/cluster-autoscaler-about.adoc

```diff
@@ -5,28 +5,42 @@
 // * osd_cluster_admin/osd_nodes/osd-nodes-about-autoscaling-nodes.adoc
 // * osd_cluster_admin/osd-cluster-autoscaling.adoc
 // * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * rosa_cluster_admin/rosa-cluster-autoscaling-hcp.adoc (temporary)
+
 
 :_mod-docs-content-type: CONCEPT
 [id="cluster-autoscaler-about_{context}"]
 = About the cluster autoscaler
 
 The cluster autoscaler adjusts the size of an {product-title} cluster to meet its current deployment needs. It uses declarative, Kubernetes-style arguments to provide infrastructure management that does not rely on objects of a specific cloud provider. The cluster autoscaler has a cluster scope, and is not associated with a particular namespace.
+ifdef::openshift-rosa-hcp[]
+In {product-title}, the Cluster Autoscaler is fully managed, which means it is hosted along with the control plane.
+endif::openshift-rosa-hcp[]
+
 
 The cluster autoscaler increases the size of the cluster when there are pods that fail to schedule on any of the current worker nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
 
+ifdef::openshift-rosa-hcp[]
+The cluster autoscaler computes the total memory, CPU, and GPU only on the nodes that belong to autoscaling machine pools. All of the machine pool nodes that are not autoscaling are excluded from this aggregation. For example, if you set the `maxNodesTotal` to `50` on a {hcp-title} cluster with three machine pools in which a single machine pool is not autoscaling, the cluster autoscaler restricts the total nodes to `50` in only those two machine pools that are autoscaling. The single manually scaling machine pool can have additional nodes, making the overall cluster nodes total more than `50`.
```

Comment:
I don't think it really makes a difference, but would it make sense to use `{product-title}` instead of `{hcp-title}` here?

Reply:
Good catch, thank you!